.processes/development.md

```diff
@@ -47,6 +47,7 @@ Representantives are elected every `3 months`, previous representatives may be r
 Responsibilities:
 
 - to be the bridge between tech team and Project Owner
+- daily check for community created tasks in [hopr-devrel](https://github.com/hoprnet/hopr-devrel)
```

Comment:
Not a big fan of such polling processes. Can we not ask people to just ping us on Element and take it from there if needed?

Reply:
will keep that for now until we revisit the comm-tech processes in a few months